src/lib/flexbox/api/base.ts

```diff
@@ -146,7 +152,7 @@ export abstract class BaseFxDirective implements OnDestroy, OnChanges {
     try {
       if (element) {
         let immediateValue = getDom().getStyle(element, styleName);
-        value = immediateValue || getDom().getComputedStyle(element).display;
+        value = immediateValue || getDom().getComputedStyle(element)[styleName];
```

Comment:
Instead of `getDom().getComputedStyle(element)[styleName]` you can use `getDom().getComputedStyle(element).getPropertyValue(styleName)`.

Reply:
As I know, in iOS 9 the `getPropertyValue` returns `null` instead of empty string as in browsers on other OS. Not sure how much people still use the iOS 9 so my comment is just for info.